poc/vdaf_poc/flp_bbcggi19.py

```diff
@@ -1147,3 +1109,65 @@ def decode(self, output: list[F], _num_measurements: int) -> int:
     def test_vec_set_type_param(self, test_vec: dict[str, Any]) -> list[str]:
         test_vec['max_measurement'] = int(self.max_measurement)
         return ['max_measurement']
+
+
+def encode_range_checked_int(
+        field: type[F],
+        value: int,
+        max_measurement: int) -> list[F]:
+    """
+    Encode an integer into multiple field elements in a way that
+    allows for efficient range proofs.
```

Comment:
I like the way you expressed this elsewhere:
```suggestion
    Encode an integer as a weighted sum of zeros and ones to allow
    for efficient range proofs.
```